grails-app/migrations/0.8.x/changelog-2023-09-21-1400-add-eventtypes-for-request-approval-workflow.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="kchelstowski" id="210920231400-1">
+        <preConditions onFail="MARK_RAN">
+            <tableExists tableName="event_type" />
+        </preConditions>
+        <insert tableName="event_type">
+            <column name="id" value="abb5b3d2-da3e-4a4c-9e3c-de139147f822"/>
+            <column name="version" valueNumeric="0"/>
+            <column name="date_created" valueDate="2023-09-21T00:00:00.0"/>
+            <column name="description" value="Request has been sent for approval"/>
+            <column name="event_code" value="PENDING_APPROVAL"/>
+            <column name="last_updated" valueDate="2023-09-21T00:00:00.0"/>
+            <column name="name" value="Waiting for approval"/>
+            <column name="sort_order" valueNumeric="1"/>
```

Comment:
I am not sure if these sort orders make sense, but I am not sure if this is currently used for anything at all.

Reply:
It's a good question. If we ever do 
```
event1 > event2
```
then we probably need to deal with it. 

But I assume we're using the ordinal value on the EventCode to do the sorting, so probably not an issue.

I'm ok leaving this for now OR changing sort order to 0.

Reply:
it seems not to be used anywhere at the moment, but since the default event types also have the sortOrder, I decided to add it.
Not to cause confusions I can either remove it (since it is nullable) or change it to 0. Let me know.